cranelift/codegen/src/timing.rs

```diff
@@ -64,6 +64,11 @@ define_passes! {
     unreachable_code: "Remove unreachable blocks",
     remove_constant_phis: "Remove constant phi-nodes",
 
+    vcode_lower: "VCode lowering",
+    vcode_post_ra: "VCode post-RA finalization",
```

Comment:
Since this is user-exposed, can you expand `RA` here, please?

Reply:
Done!